authentication/AuthenticationExample/ViewControllers/AccountLinkingViewController.swift

```diff
@@ -139,16 +139,18 @@ class AccountLinkingViewController: UIViewController, DataSourceProviderDelegate
     guard let clientID = FirebaseApp.app()?.options.clientID else { return }
 
     // Create Google Sign In configuration object.
+    // TODO: Move configuration to Info.plist
```

Comment:
This value is being pulled from the plist file; what's the improvement to be made here?

Reply:
It's the Info.plist (not GoogleService-Info.plist) as specified by the new Google SignIn 7 configuration docs.